aarapsi_robot_pack/RequestSVM params
bool success
